src/app/package.json

```diff
@@ -11,6 +11,7 @@
         "core-js": "2.6.4",
         "file-saver": "2.0.0",
         "immutability-helper": "2.9.0",
+        "json2csv": "4.1.6",
```

Comment:
This library's no longer used so I dropped it in #237. Consider removing it before merging.

Reply:
Oh good catch. Removing it now.